release/rancher/rancher.go

```diff
@@ -27,49 +24,80 @@ const (
 	rancherImagesFileName    = "/rancher-images.txt"
 	rancherHelmRepositoryURL = "https://releases.rancher.com/server-charts/latest/index.yaml"
 
-	setKDMBranchReferencesScriptFile = "set_kdm_branch_references.sh"
-	setKDMBranchReferencesScript     = `#!/bin/bash
-set -x
+	setKDMBranchReferencesScriptFileName = "set_kdm_branch_references.sh"
+	setChartReferencesScriptFileName     = `set_chart_references.sh`
+	cloneCheckoutRancherScript           = `#!/bin/bash
```

Comment:
Are we using bash specific functionality? If not, can you change this to `#!/bin/sh`? Shellcheck should confirm POSIX compliance.